mundane-csv/src/main/scala/com/ambiata/mundane/csv/SimpleCsv.scala

```diff
@@ -28,14 +28,23 @@ case class SimpleCsv(delimiter: Char, quoteCharacter: Option[Char], escapeCharac
 
   def parse(line: String): String \/ List[String] =
     \/.fromTryCatchNonFatal {
-      if (line.isEmpty) List("")
+      if (line.isEmpty) emptyLineDefault
```

Comment:
This feels unnecessary. Alternatively could you not always return `Nil` and then consumers can do what they like. (eg. `.map(l => if (l.isEmpty) List("") else l)`).